teachopencadd/talktorials/T013_query_pubchem/talktorial.ipynb

```diff
@@ -9,7 +9,8 @@
     "Authors:\n",
```

Comment:
<p>Wonderful, I like the new function split very much!</p><p> </p><p>Minor comment w.r.t. to some of your docstrings: I only know the format without leading colons for the return values and a space between the "Parameters" and "Returns" block.</p><p> </p><p><code>Returns</code></p><p><code>-------</code></p><p><code>list</code></p><p><code>&nbsp;&nbsp;&nbsp;&nbsp;The PubChem CIDs of similar compounds.&nbsp;</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/81/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
<p>thanks, I fix the docstring.</p>